### ISSUE TITLE
onnx error

### ISSUE THREAD

**Issue by cqray1990** [ouyanghuiyu/chineseocr_lite#196]
import sys
#import mxnet as mx

sys.path.insert(0, "..")
from crnn import LiteCrnn
from utils0nnx import load_model
import torch
from angle_class import shufflenet_v2_x0_5
from crnn.keys import alphabetChinese as alphabet

nh = 256
net   =LiteCrnn(32, 1, len(alphabet) + 1, nh, n_rnn=2, leakyRelu=False, lstmFlag=True)
#net = shufflenet_v2_x0_5(num_classes=4, pretrained=False)

net = load_model(net, '../models/crnn_lite_lstm_dw.pth')

net.eval()

print('Finished loading model!')
print(net)
device = torch.device("cpu")
net = net.to(device)

##################export###############
output_onnx = 'crnn_lite_lstm_dw.onnx'
print("==> Exporting model to ONNX format at '{}'".format(output_onnx))
input_names = ["input"]
# output_names = ["hm" , "wh"  , "reg"]
output_names = ["out"]
inputs = torch.randn(1, 1, 32, 196).to(device)
torch_out = torch.onnx._export(net, inputs, output_onnx, export_params=True, verbose=False,
                               input_names=input_names, output_names=output_names)

import os

os.system("python3 -m onnxsim {0} {0}".format(output_onnx))

i  run this code to convert to onnx but it raise Simplifying...
Segmentation fault (core dumped):

but i when i convert shufflenetv2_05.pth, there no problem, can you tell me why? thank you very much :

torch:1.5.1
cuda:10.1
onnx                 1.7.0
onnx-simplifier      0.2.9
onnx-tf              1.5.0
onnxruntime          1.3.0
